aerogear/data-sync-ui
# Review styles to match given mockups.

## Motivation
The UI should look as much as possible like the mockups given by design/UX responsibles.

## What

- Reviews CSS and change what's necessary.
- Moves styles from main.css into specific .css files per component, when possible.
- Refactors how classes are assigned into a more JS friendly way (from `styles["class"]` to styles.class)
- Refactors in resolvers list components, for simplicity and easy of readability.

## Verification Steps

- Verify UI resembles [mockups](https://redhat.invisionapp.com/share/SVMB0QW6MR9#/screens) as much as possible. 
- Due to refactorizations, verify Resolvers tab keeps working as expected.
